lib/simple_url_preview.dart

```diff
@@ -22,18 +22,27 @@ class SimpleUrlPreview extends StatefulWidget {
   /// Whether or not to show close button for the preview
   final bool isClosable;
 
-  /// Text color
+  /// Text color. Ignored for any item where a style is specified
   final Color textColor;
```

Comment:
As we are adding text, description styles, I think we can remove this property.

Reply:
This would be a breaking change, yes? Figured I'd keep backwards compatibility so you don't need a major version increase. Thoughts?

Reply:
I think we should be okay with breaking change, as its only single property that needs to be removed. We can mention that in Readme.

Reply:
Any chance you'd be willing to accept the change as it is, and then introduce the breaking change in a separate commit?

Reply:
Yes sure.. I will add that breaking change in the next commit and make the release..